microsoft-edge/extensions-chromium/developer-guide/devtools-extension.md

```diff
@@ -0,0 +1,430 @@
+---
+title: Create a DevTools extension
+description: In this tutorial, you learn to build a Microsoft Edge extension that extends DevTools. At the end of the tutorial, you should have a working DevTools extension that creates its own panel and interacts with the DevTools extension APIs.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: extensions
+ms.date: 9/27/2022
+---
+# Create a DevTools extension
+
+This tutorial shows you how to build your own DevTools extension from scratch. It is a good way to experience everything that is needed for typical development. At the end of the tutorial, you should have a working extension that creates its own panel and interacts with the DevTools APIs.
+
+First please review the prerequisites. Then download and run the final extension from the [Solution](#solution) section to see what you will be building in this tutorial. Otherwise, start at the [Step 1 - Create a Basic DevTools Extension](#step-1---create-a-basic-devtools-extension) section to build the extensions from scratch.
+
+
+<!-- ====================================================================== -->
+## Prerequisites
+
+* [Microsoft Edge Browser](https://www.microsoft.com/edge) to run the extension.
+* A code editor such as [Visual Studio Code](https://code.visualstudio.com/) to follow along with the tutorial.
+* Knowledge of [GitHub](https://github.com/) to download the code presented in this tutorial.
+
+
+<!-- ====================================================================== -->
+## Solution
+
+In this section, you download and run the final state of the extension that the rest of this tutorial will teach you to build. Later, you will start from scratch and learn to write the code to create your own extension.
+
+1. Get the final extension code by [downloading this zip file](https://codeload.github.com/MicrosoftEdge/Demos/zip/refs/heads/main) and extracting its content to a folder on your computer.
+
+1. Open Microsoft Edge and navigate to the `edge://extensions/` page.
+
+1. Make sure that **Developer mode** is enabled.
```

Comment:
```suggestion
1. Turn on the **Developer mode** toggle.
```
off for me by default.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle